frontend/src/admin/admin-router.tsx

```diff
@@ -9,7 +10,11 @@ export function AdminRouter() {
   return (
      <Routes>
        <Route path={AdminRoute.EDIT} element={<div> Edit </div>}/>
-       <Route path={AdminRoute.HOME} element={<div> Admin </div>}/>
+       <Route
+           path={AdminRoute.HOME}
+           element={
+           <UserList/>
+       }/>
```

Comment:
This would be nice to have on a single line for readability:
```suggestion
           element={<UserList/>}
       />
```

Reply:
✅